src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelMetrics.cs

```diff
@@ -18,6 +19,8 @@ internal sealed class KestrelMetrics
     // Note: Dot separated instead of dash.
     public const string MeterName = "Microsoft.AspNetCore.Server.Kestrel";
 
+    public const string ErrorType = "error.type";
```

Comment:
I might have called this "ErrorTypeAttributeName", but I don't feel strongly about it.